src/components/MainPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./MainPage.css";
+
+function MainPage() {
+  return (
+    <div className="main-container">
+      <div className="logo-container">
+        <img src="src/images/bee.png"></img>
+        <img src="src/images/hive_logo.png"></img>
+        <img src="src/images/logo2.png"></img>
+      </div>
+      <div className="btn-container">
+        <button className="btn" id="btn-play">
+          <Link to="/play" id="btn-play-link">
+            PLAY
+          </Link>
```

Comment:
<Link> component should wrap outside the button, not the text

Reply:
Fixed